### DISCUSSION TITLE
update acap from axis command line

### DISCUSSION THREAD

**Discussion by rafiomeon** [AxisCommunications/discussions#87] [ACAP] [answered]
I used to update axis acap file from axis camera webpage by removing older version and install newer version. I have a scenario where the Axis Camera webpage is not accessible and I have to update the current package using camera ssh terminal. Is it possible to update the current acap package from the axis camera terminal.

Thanks,
Rafi

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @rafiomeon. To install or update an ACAP app you can use the [`upload.cgi` in VAPIX](https://www.axis.com/vapix-library/subjects/t10102231/section/t10036126/display?section=t10036126-t10010609), it does not require you to ssh into the device.

**Reply by haichaoji123** [reply]
https://www.axis.com/vapix-library/subjects/t10102231/section/t10036126/display
